src/Microsoft.MobileBlazorBindings.WebView/BlazorHybridRenderer.cs

```diff
@@ -51,6 +58,65 @@ public BlazorHybridRenderer(IPC ipc, IServiceProvider serviceProvider, ILoggerFa
             CaptureAsyncExceptions(initTask);
         }
 
+        /// <summary>
+        /// Signals that a render batch has completed.
+        /// </summary>
+        /// <param name="incomingBatchId">The render batch id.</param>
+        /// <param name="errorMessageOrNull">The error message or null.</param>
+        /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
+        public Task OnRenderCompletedAsync(long incomingBatchId, string errorMessageOrNull)
+        {
+            if (this._disposing)
```

Comment:
Not a big deal right now, but we avoid `this.`. I thought the editorConfig in the file might catch this but I don't recall. Feel free to update this, or I will eventually do another cleanup pass in the future anyway (and probably break something while I'm at it).